sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -138,7 +138,7 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
    */
   @throws[AnalysisException]("table does not exist")
   override def listColumns(tableName: String): Dataset[Column] = {
```

Comment:
I made mistakes too much. Sorry for that.

I will change like the following.
```scala
/**
 * Returns a list of tables in the current database.
 * This includes all temporary tables.
 *
 * @since 2.0.0
 */
def listTables(): Dataset[Table]
```